ocs_ci/utility/utils.py

```
@@ -6769,3 +6769,84 @@ def create_kubeconfig(kubeconfig_path):
+
+def get_current_git_branch():
+    """                                                                                                                                                                                                                                           
+    Get the current git branch name of ocs-ci repository.                                                                                                                                                                                         
+                                                                                                                                                                                                                                                    
+    Returns:                                                                                                                                                                                                                                      
+        str: Branch name (e.g., 'master', 'release-4.18')                                                                                                                                                                                         
+    """                                                                                                                                                                                                                                           
+    try:        
+        # Try to get branch from git                                                                                                                                                                                                              
+        result = subprocess.run(                                                                                                                                                                                                                  
+            ['git', 'rev-parse', '--abbrev-ref', 'HEAD'],                                                                                                                                                                                         
```

Ruff (0.15.12)

[error] Starting a process with a partial executable path

(S607)